talk/owt/sdk/base/dualvideoencoder.h

```diff
@@ -0,0 +1,30 @@
+#ifndef OWT_BASE_DUALVIDEOENCODER_H_
+#define OWT_BASE_DUALVIDEOENCODER_H_
+
+#include <vector>
+
+#include "webrtc/api/video_codecs/sdp_video_format.h"
+#include "webrtc/api/video_codecs/video_encoder.h"
+#include "webrtc/api/video_codecs/video_encoder_factory.h"
+
+namespace owt {
+namespace base {
+
+
+class DualVideoEncoder : public webrtc::VideoEncoderFactory {
```

Comment:
This is a key abstraction in the base WebRTC layer. This factory gets created once a the start of the application and `CreateVideoEncoder()` is called for each stream to encode.